inc/lang/pt/lang.php

```diff
@@ -3,7 +3,7 @@
 /**
  * @license    GPL 2 (http://www.gnu.org/licenses/gpl.html)
  *
- * @author Jose Luis Sanches <jose.reis.sanches@gmail.com>
+ * @author Eduardo Mozart de Oliveira <eduardomozart182@gmail.com>
```

Comment:
Unexpected output of the translation tool: it should keep the other author.
Maybe it is because the DokuWiki repository used by the tool is a bit older.